### PR TITLE
A standard is not always encoded as a single document

### PR DESCRIPTION
**Pull request by kriswest**
... and might instead be formed by a collection of web pages and other artifacts such as source code files.

We've been advised (on the FINOS/FDC3 standard project) that although the convention used by ANSI, ISO and ECMA is for a standard to be encoded as a single document, this is not the case for other organisations such the W3C or IETF (among others), which are modern and web-based. This is also the case for FDC3 where the standard is formed by a number of webpages (forming part of the same website), supported by source code and schema files. 

However, there is language in the CSL 1.0 (specifically in the governance section) that specifically refers to a single document. This PR seeks modify that language to allow it to apply to other artifacts such as webpages and source code files that are developed under and governed by the Community Specification.

We could of course make these changes in our own copy of these documents, however, we both desire to remain aligned with the CSL and are interested in the opinion of the CSL's maintainers on the change.

### CODE REVIEW

File: 5._Governance.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Community Specification Governance Policy 1.0
 
-This document provides the governance policy for specifications and other documents developed using the Community Specification process in a repository (each a “Working Group”).  Each Working Group and must adhere to the requirements in this document.
+This document provides the governance policy for specifications and other documents or artifacts, such as a web page or source code, developed using the Community Specification process in a repository (each a “Working Group”).  Each Working Group and must adhere to the requirements in this document.
```
Comment (kriswest) [inline 5._Governance.md]:
```suggestion
This document provides the governance policy for specifications and other documents or artifacts, such as web pages or source code or schema files, developed using the Community Specification process in a repository (each a “Working Group”).  Each Working Group and must adhere to the requirements in this document.

```

File: 5._Governance.md [hunk 2]
```diff
@@ -36,9 +36,9 @@ Inspired by [ANSI’s Essential Requirements for Due Process](https://share.ansi
 
 ## 4.	Specification Development Process.  
 
-**4.1.	Pre-Draft.**  Any Participant may submit a proposed initial draft document as a candidate Draft Specification of that Working Group.  The Maintainer will designate each submission as a “Pre-Draft” document.
+**4.1.	Pre-Draft.**  Any Participant may submit a proposed initial draft document or other suitable artifact as a candidate Draft Specification of that Working Group.  The Maintainer will designate each submission as a “Pre-Draft”.
```
Comment (kriswest) [inline 5._Governance.md]:
```suggestion
**4.1.	Pre-Draft.**  Any Participant may submit a proposed initial draft document or other suitable artifact or artifacts as a candidate Draft Specification of that Working Group.  The Maintainer will designate each submission as a “Pre-Draft”.

```

File: 5._Governance.md [hunk 3]
```diff
@@ -36,9 +37,10 @@ Inspired by [ANSI’s Essential Requirements for Due Process](https://share.ansi
 
 ## 4.	Specification Development Process.  
 
-**4.1.	Pre-Draft.**  Any Participant may submit a proposed initial draft document as a candidate Draft Specification of that Working Group.  The Maintainer will designate each submission as a “Pre-Draft” document.
+**4.1.	Pre-Draft.**  Any Participant may submit a proposed initial draft document or other suitable artifact or artifacts as a candidate Draft Specification of that Working Group.  The Maintainer will designate each submission as a “Pre-Draft”.
+
 
-**4.2.	Draft.**  Each Pre-Draft document of a Working Group must first be Approved to become a” Draft Specification”.  Once the Working Group approves a document as a Draft Specification, the Draft Specification becomes the basis for all going forward work on that specification.
+**4.2.	Draft.**  Each Pre-Draft of a Working Group must first be Approved to become a "Draft Specification".  Once the Working Group approves a document or artifact as a Draft Specification, the Draft Specification becomes the basis for all going forward work on that specification.
```
Comment (hughtroeger) [inline 5._Governance.md]:
```suggestion
**4.2.	Draft.**  Each Pre-Draft of a Working Group must first be Approved to become a "Draft Specification".  Once the Working Group approves a document or artifact as a Draft Specification, the Draft Specification becomes the basis for all work on that specification going forward.

```

**Review by hughtroeger — APPROVED** [review]

**Comment by mindthegab** [comment]
Hey @mkdolan @scooter99boston @jpradocueva - any thoughts about whether this will make into the CSL and a timeline? Or should we consider proposing this change downstream in the [FINOS CSL fork](https://github.com/finos/standards-project-blueprint/)?

FDC3 (fdc3.finos.org) will be adopting the CSL soon, so it would be important to identify an approach here before they do.

Thanks in advance!

**Comment by mkdolan** [comment]
> Hey @mkdolan @scooter99boston @jpradocueva - any thoughts about whether this will make into the CSL and a timeline? Or should we consider proposing this change downstream in the [FINOS CSL fork](https://github.com/finos/standards-project-blueprint/)?
> 

I would propose it downstream first and then we can evaluate in our next CSL review but changing CSL will definitely take longer.

**Comment by mindthegab** [comment]
Thanks @mkdolan - I’ll connect with the contributor and discuss options
downstream.

**Comment by mindthegab** [comment]
@kriswest Per our convo [here](https://github.com/finos/FDC3/issues/412#issuecomment-1113561893), probably makes sense to open the issue downstream at https://github.com/finos/standards-project-blueprint. 

Let me know if you want me to do it and if so, maybe we should close this issue until a decision is made at FINOS level.

**Comment by mindthegab** [comment]
@greyseer256 this issue can be closed for now.

**Comment by kriswest** [comment]
@mindthegab should we leave it open so it makes the agenda for the next CSL review? I'll happily update it when change the version of this PR you commented on downstream.

**Comment by mindthegab** [comment]
@kriswest I think up to the maintainers @greyseer256, but definitely sounds like a good idea @kriswest on second thoughts.

File: 5._Governance.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # Community Specification Governance Policy 1.0
 
-This document provides the governance policy for specifications and other documents developed using the Community Specification process in a repository (each a “Working Group”).  Each Working Group and must adhere to the requirements in this document.
+This document provides the governance policy for specifications and other documents or artifacts, such as web pages or source code or schema files, developed using the Community Specification process in a repository (each a “Working Group”).  Each Working Group and must adhere to the requirements in this document.
```
Comment (kriswest) [inline 5._Governance.md]: removed reference to source code
```suggestion
This document provides the governance policy for specifications and other documents or artefacts, such as web pages, schemas or interface definitions developed using the Community Specification process in a repository (each a “Working Group”). Each Working Group must adhere to the requirements in this document.

```

**Comment by kriswest** [comment]
Discussed this today with @scooter99boston and @robmoffat and understand that the change proposed would be difficult. For FDC3s purposes the abstract page for the standard version (which links to the rest of the documents, all held in the same repository) can be used as the document and linked to at specific versions via a github tag.

Hence, I'll go ahead and close this.

**Pull request closed without merging**